src/model/element/complex.js

```diff
@@ -57,6 +57,21 @@ class Complex extends Entity {
     return this.elementSet.create();
   }
 
+  name( newName ){
+    if( newName !== undefined ){
+      return this.rename( newName );
+    } else {
+      let name;
+      if( this.named() ){
+        name = this.syncher.get('name');
```

Comment:
```suggestion
        name = super.name();
```

Reply:
You should delegate the low-level getter to the superclass so that whatever logic within the superclass in maintained -- e.g. there could be general name post-processing rules within `Entity` that apply to all node types.